### PR TITLE
Functional CSS take 2

### PR DESCRIPTION
**Pull request by brousalis**
Redid the files to match our map patterns and .sass files, rem'd everything.

I tried to avoid using maps for the sake of using maps. I'd like some opinions on the margin/padding files. 

I basically used 

``` scss
$positions: (
  (top,    t)
  (right,  r)
  (bottom, b)
  (left,   l)
);
```

and a few other shortcuts.

I left out the README so it didnt clog up the PR. I'll add that later. The compile size is now about 8kb for the entire class (down 1kb, 🔑 ). Again it's not included by default, so won't affect anyone who doesn't want it.

@shayhowe @ashtonthomas @DLavin23

### CODE REVIEW

File: vendor/assets/stylesheets/rolodex/settings/utilities/lib/_margin.sass [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+
+  resets:
+     .m-0
+    .mt-0
+    .mr-0
+    .mb-0
+    .ml-0
+*/
+
+// Margin from $baseline variables
```
Comment (brousalis) [inline vendor/assets/stylesheets/rolodex/settings/utilities/lib/_margin.sass]: we could instead do this like:

```scss
.mt0 - 0
.mt1 - $baseline-steo
.mt2 - $baseline-x-small
.mt3 - $baseline-small
.mt4 - $baseline-base
.mt5 - $baseline-large
.mt6 - $baseline-x-large
.mt7 - $baseline-xx-large
```

I think I prefer the `-baseline` way however

File: vendor/assets/stylesheets/rolodex/components/_grid-responsive.sass [hunk 2]
```diff
@@ -0,0 +1,28 @@
+// Responsive Grid
+// ========================================
+
+// Responsive grid classes from $breakpoints
```
Comment (shayhowe) [inline vendor/assets/stylesheets/rolodex/components/_grid-responsive.sass]: Is there a version of the grid that isn't within a breakpoint? For example, does `.grid-1-2` exist if I want 50% all of the time?

Comment (shayhowe) [inline vendor/assets/stylesheets/rolodex/components/_grid-responsive.sass]: Maybe that is in another file? To which, should this grid and that grid be within the same file?

File: vendor/assets/stylesheets/rolodex/settings/utilities/lib/_clearfix.sass [hunk 3]
```diff
@@ -0,0 +1,29 @@
+// Clearfix
+// ========================================
+
+.clearfix:before,
+.clearfix:after,
+.group:before,
+.group:after
+  content: " "
```
Comment (shayhowe) [inline vendor/assets/stylesheets/rolodex/settings/utilities/lib/_clearfix.sass]: I'd remove the space between the quotation marks.

Comment (shayhowe) [inline vendor/assets/stylesheets/rolodex/settings/utilities/lib/_clearfix.sass]: Also, why have `clearfix` and `group`?

File: vendor/assets/stylesheets/rolodex/settings/utilities/lib/_layout.sass [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+.overflow-scroll
+  overflow: scroll
+
+.overflow-auto
+  overflow: auto
+
+// Floats
+
+.pull-left
+.float-left,
```
Comment (shayhowe) [inline vendor/assets/stylesheets/rolodex/settings/utilities/lib/_layout.sass]: Why use `pull` and `float`?

File: vendor/assets/stylesheets/rolodex/settings/utilities/lib/_margin.sass (hunk 1 above)
Comment (shayhowe) [inline vendor/assets/stylesheets/rolodex/settings/utilities/lib/_margin.sass]: I'm a baseline fan for sure.

**Comment by shayhowe** [comment]
@brousalis Let's chat when you get it, I'd like to poke around the folder structure and see if I can make sense of how everything is organized.

File: vendor/assets/stylesheets/rolodex/components/_grid-responsive.sass (hunk 2 above)
Comment (brousalis) [inline vendor/assets/stylesheets/rolodex/components/_grid-responsive.sass]: this is an optional grid, grid-responsive, not included by default. the basic grid is included by default. I just moved it from the `utilities` folder to `components/grid`

File: vendor/assets/stylesheets/rolodex/settings/utilities/lib/_clearfix.sass (hunk 3 above)
Comment (brousalis) [inline vendor/assets/stylesheets/rolodex/settings/utilities/lib/_clearfix.sass]: group is a standard rolodex set. however, search the bellycard repos and you'll find tons of instances of `.clearfix`. supporting both.

File: vendor/assets/stylesheets/rolodex/settings/utilities/lib/_layout.sass (hunk 4 above)
Comment (brousalis) [inline vendor/assets/stylesheets/rolodex/settings/utilities/lib/_layout.sass]: same thing as clearfix/group

**Comment by ashtonthomas** [comment]
@shayhowe @brousalis did anything come from your offline discussion related to this?

**Comment by brousalis** [comment]
take 3 is coming

**Pull request closed without merging**